app/assets/stylesheets/style.scss

```diff
@@ -328,4 +350,45 @@ label:hover {
   }
 }
 
+#footer {
+position: fixed;
```

Comment:
Line should be indented 2 spaces, but was indented 0 spaces<br>Properties should be ordered background-color, border-top, bottom, height, margin-top, position, width